plugins/frontend/backstage-plugin-jira/src/api/index.ts

```diff
@@ -217,4 +229,33 @@ export class JiraAPI {
       ),
     ];
   }
+
+  async getIssuesCountByJql(query: CustomQuery) {
+    const { apiUrl } = await this.getUrls();
+
+    // TODO : mutualize this ?
+    const request = await fetch(`${apiUrl}search`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        jql: query.query,
```

Comment:
A co-worker of mine suggested that we filter out "PROJECT = XXXX" from the JQL queries, to avoid requesting data from another project than the one specified in the entity annotation ` jira/project-key: <example-jira-project-key>`

Reply:
Yes, I think that we should append `PROJECT=${projectFromAnnotation} AND` for each query; and add a check on `query` value to throw an error if we found a `PROJECT=` into it.